login-workflow/example2/src/screens/new-architecture-test-screens/LoginScreen.tsx

```diff
@@ -26,42 +26,49 @@ export const LoginScreen = (): JSX.Element => {
                         <Box>
                             <Button
                                 variant="contained"
-                                sx={{ width: 200 }}
+                                sx={{ width: 180 }}
                                 onClick={(): void => setIsAuthenticated(true)}
                             >
                                 Login
                             </Button>
                             <Button
                                 variant="contained"
-                                sx={{ width: 200, ml: 2 }}
+                                sx={{ width: 180, ml: 2 }}
```

Comment:
Thanks for making these fit better. It doesn't matter if the buttons collide at this point as this is just a test environment for now. We will eventually stat moving things into a working app where we won't need these placeholder buttons. All that to say, don't spend too much time cleaning up demo pieces that aren't published in the end.